bluemira/equilibria/coils/_grouping.py

```diff
@@ -322,18 +334,20 @@ def from_group_vecs(cls, eqdsk: EQDSKInterface) -> CoilGroup:
                         current=0,
                         dx=dx,
                         dz=dz,
-                        ctype="NONE",
+                        ctype=ct or CoilType.NONE,
+                        name=cn,
                     )
                 )
-            elif dx != dz:  # Rough and ready
+            elif ct is CoilType.CS or (dx != dz and ct is None):  # Rough and ready
                 cscoils.append(
                     Coil(
                         eqdsk.xc[i],
                         eqdsk.zc[i],
                         current=eqdsk.Ic[i],
                         dx=dx,
                         dz=dz,
-                        ctype="CS",
+                        ctype=ct or CoilType.CS,
```

Comment:
```suggestion
                        ctype=CoilType.CS,
```

Can only be `CS` here right?

Reply:
Or `None` but I think we want the ctype to be `CS` here